react/UploadQueue/Readme.md

```diff
@@ -51,7 +52,7 @@ const data = {
 };
 
 <>
-  popover: <input type="checkbox" value={state.popover} onChange={() => setState({ popover: !state.popover })} />
+  popover: <input type="checkbox" value={state.popover} onChange={() => setState({ popover: !state.popover })}/>
```

Comment:
nit: on pourrait utiliser le comp checkbox plutôt qu'un input, même si ce n'est que de la doc

Reply:
i won't do it. 

🙏 next time, please provide a suggestion to help solving the review (especially after a third review)